products/redhat-openjdk.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Red Hat OpenJDK
```

Comment:
_Red Hat build of OpenJDK_ looks more common. If changed, should be changed in the filename, permalink and in the title / description.